sig/shims/prism.rbs

```diff
@@ -0,0 +1,13 @@
+module Prism
```

Comment:
The `prism` gem has a `sig/` directory, but it was missing the `Prism::Translation::Parser33` class that `steep` uses.